packages/itwin/ec3-widget/src/components/EC3/EC3Localization.ts

```diff
@@ -0,0 +1,37 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+export interface EC3LocalizationResult {
```

Comment:
With this interface consumer will need to provide ALL of the strings instead of just the ones they want to override. This interface should not exist. A simple map should be enough. Map key - localization key, Map value - translated string. Some type safety will be lost but this might be easier to document. Document the map and point to the provided `locales/en/ec3Widget.json` for all of the available keys.

Reply:
done